sdk/purestorageblock/Azure.ResourceManager.PureStorageBlock/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### Other Changes
 
+## 1.0.0 (2025-06-26)
```

Comment:
you need to remove the "Unreleased" section.
Also please update the version number in our csproj file to be 1.0.0